app/gui/view/component-browser/searcher-list-panel/src/layout.rs

```diff
@@ -0,0 +1,340 @@
+//! A module with description how the list of Component Groups is laid out in the Component Panel
+//! List.
+
+use crate::prelude::*;
+
+use ensogl_grid_view::Col;
+use ensogl_grid_view::Row;
+use ide_view_component_group::set::GroupId;
+use ide_view_component_group::set::SectionId;
+
+
+
+// =================
+// === Constants ===
+// =================
+
+/// Height of the header of the component group. This value is added to the group's number of
+/// entries to get the total height.
+pub const HEADER_HEIGHT: usize = 1;
```

Comment:
What unit is this? Are these pixels? Oh this are elements? Maybe the name should be `HEADER_OVERFLOW_ELEM_COUNT` or something like that?